docs/gc_vgclog_tags.md

```diff
@@ -0,0 +1,70 @@
+# vgc log XML tags and attributes
```

Comment:
I think we should have this file as a reference section at the bottom of the file that contains verbose GC logging content.

Reply:
Totally agree - that's what is currently sitting on my other PR :)